src/model/bioprotocol.v1.yaml

```diff
@@ -0,0 +1,25 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: BioProtocol
+type: array
+items:
+    type: object
+    properties:
+        protocol_sequencing_number:
+            description: "section ID, looks like 's4-1'"
+            type: string
+        protocol_title:
+            description: "section title, looks like 'Isolation of LDs and treatments'"
+            type: string
+        is_protocol:
+            type: boolean
```

Comment:
always `false` in the example, what does it mean? An enumeration can be more extensible than a boolean e.g. `type`: `protocol`|`something-else` allows addition of other types.

Reply:
this is a flag they use when they determine that a protocol that has been given to them isn't actually a protocol - eLife mixes materials and methods together in the same section with no means of differentiation, and there is some talk at the moment on how to best deal with that.

```
$ curl https://dev.bio-protocol.org/api/elife00005 | jq '.Protocols[].IsProtocol'`
true
true
true
true
true
true
```

I'll clarify with them, but I think what I'll do is exclude rows from being returned to the journal where `IsProtocol=false` so that a link isn't generated. And I'll exclude this field as well as it will then be redundant.

Reply:
I've pinged Dong about it. It seems the most obvious and reasonable thing to do and they seem quite competent so I'll go ahead and make the changes. I can always revert if they start going crazy.

Reply:
yes, Dong confirmed this is what is happening